### PR TITLE
Feat/add accordion

### PR DESCRIPTION
**Pull request by Yanhuazong**
-Reconfigured plugin with wp-scripts
-add accordion block

Thanks!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,12 +3,31 @@
...
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
+    "bulma": "^0.8.2"
+  },
+  "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.10.4",
+    "@wordpress/scripts": "^12.1.1",
+    "ignore-emit-webpack-plugin": "^2.0.2"
```
Comment (jyoung295) [inline package.json]: Why did you add all these packages, everything should be included in wp-scripts.

File: package.json [hunk 2]
```diff
@@ -3,12 +3,31 @@
...
+    "format:js": "wp-scripts format-js",
+    "lint:css": "wp-scripts lint-style",
+    "lint:js": "wp-scripts lint-js",
+    "lint:md:docs": "wp-scripts lint-md-docs",
+    "lint:md:js": "wp-scripts lint-md-js",
+    "lint:pkg-json": "wp-scripts lint-pkg-json",
+    "packages-update": "wp-scripts packages-update",
+    "start": "wp-scripts start",
+    "test:e2e": "wp-scripts test-e2e",
+    "test:unit": "wp-scripts test-unit-js"
```
Comment (jyoung295) [inline package.json:18]: We don't need all of these, just the `start` and `build` scripts.

**Comment by jyoung295** [comment]
Also looks like there are conflicts, but it says they can't be resolved in the web editor, so you'll need to delete the old build files and find the conflicts in blocks.js and gitignore.

File: package.json (hunk 1 above)
Comment (Yanhuazong) [reply package.json]: They were removed in a later commit.

File: package.json (hunk 2 above)
Comment (Yanhuazong) [reply package.json:18]: Okay. I'll remove them.

**Comment by Yanhuazong** [comment]
Yes. I'll resolve them. Does everything else look okay? I feel the paddings look a little too big if there are a group of them. 
Screen Shot 2020-08-18 at 11 21 03 AM

File: package.json (hunk 1 above)
Comment (jyoung295) [reply package.json]: Oh got it, I must have looked at the wrong thing, thanks!

**Comment by jyoung295** [comment]
> I feel the paddings look a little too big if there are a group of them.

Do you mean the top and bottom padding inside the border?

**Comment by Yanhuazong** [comment]
Yes.

**Comment by jyoung295** [comment]
Okay looks good now

**Comment by Yanhuazong** [comment]
May I merge it to master now?
